front/style/pages/_grande-marche-europe.scss

```diff
@@ -0,0 +1,287 @@
+            @media (max-width: $mobile) {
+                font-size: 3em;
+            }
+        }
+    }
+    /*FOOTER*/
+    footer {
+        position: fixed;
+        bottom: 0;
+        width: 100%;
+        display: flex;
+        justify-content: space-around;
+        box-sizing: border-box;
+        animation-timing-function: ease-in-out;
+        transition: all 2s;
+        transform: translateY(100%);
+        z-index: 4;
+        input,
+        button {
+            box-sizing: border-box;
+            width: 100%;
+            border: transparent;
+            padding: 1em 0;
+        }
+         ::placeholder {
+            color: $purple;
+        }
+        input {
+            font-size: 1.5em;
+            color: $purple;
+            padding-left: 2em;
+            width: 70%;
+            @media (max-width: $mobile) {
+                font-size: 0.8em;
+            }
+        }
+        button {
+            background-color: $purple;
+            color: #FFF;
+            cursor: pointer;
+            font-size: 1.5em;
+            transition: all 2s;
+            width: 30%;
+            &:hover {
+                transition: all 2s;
+                border-bottom: 10px solid #FFF;
+                box-shadow: inset 0 -5px 0 #FFF;
+            }
+            @media (max-width: $mobile) {
+                font-size: 0.8em;
+            }
+        }
+    }
+}
+
+
+
+.second-part {
+    display: flex;
+    width: 100%;
+    @media (max-width: $tablet) {
+        flex-direction: column;
+    }
+    .left-part {
+        background-color: $yellow;
+        height: 100vh;
+        width: 50%;
+        /*padding: 3em;*/
+        transition: all 2s;
+        z-index: 3;
+        @media (max-width: $tablet) {
+            width: 100%;
+            padding: 0;
+            height: 100vh;
+        }
+        .left-part-content {
+            margin-top: 50vh;
+            transform: translateY(-50%);
+        }
+        p {
+            top: 20%;
+            color: $darker;
+            font-size: 2em;
+            @media (max-width: $mobile) {
+                font-size: 1.5em;
+            }
+        }
+        .author {
+            font-size: 0.8em;
+            letter-spacing: 2px;
+        }
+    }
+    .right-part {
+        height: 100vh;
+        width: 50%;
+        transition: all 2s;
+        z-index: 3;
+        background-color: #FFF;
+        @media (max-width: $tablet) {
+            width: 100%;
+            padding: 0;
+            height: 50vh;
+        }
+        p {
+            font-size: 2em;
+            color: $pink;
+            top: 20%;
+        }
+        button {
+            margin-top: 50vh;
+            transform: translateY(-50%);
+            padding: 1em 2em;
+            @media (max-width: $tablet) {
+                margin: 10vh 0;
+                transform: translateY(0);
+            }
+        }
+    }
+    .quote-photo {
+        background-image: url('/images/casta.jpg');
```

Comment:
Use the implementation that we use elsewhere that allows for faster loading of the images such as here for instance

https://github.com/EnMarche/en-marche.fr/blob/57b674917d2dbe3748244213fe488b0430360eb7/templates/page/elles-marchent.html.twig#L15